finalProject/tenSpeckledFrogs.js

```diff
@@ -0,0 +1,25 @@
+var startFrogs = ["3", "2", "1"];
+var endFrogs = ["then there were 2 speckled frogs.",
+"then there was 1 speckled frog.",
+"then there were no more speckled frogs!"];
+var rhyme = " speckled frogs sat on a log eating some most delicious bugs. One jumped in the pool where its nice and cool,"
+
+for (var x = 0; x < startFrogs.length + endFrogs.length; x++) {
```

Comment:
I don't think you need to add together the length of `startFrogs` and the length of `endFrogs` - did you want this loop to run 6 times?